examples/digital_fingerprinting/production/morpheus/dfp/modules/dfp_preprocessing.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import pickle
+import time
+
+import mrc
+from mrc.core import operators as ops
+
+from morpheus.utils.column_info import process_dataframe
+from morpheus.utils.module_utils import get_module_config
+from morpheus.utils.module_utils import register_module
+
+from ..messages.multi_dfp_message import MultiDFPMessage
+
+logger = logging.getLogger(f"morpheus.{__name__}")
+
+
+@register_module("DFPPreprocessing", "morpheus_modules")
+def dfp_preprocessing(builder: mrc.Builder):
```

Comment:
I think we want a different name for this stage to differentiate it from `dfp_pipeline_preprocessing`. Maybe `DFPDataPrep`?